### DISCUSSION TITLE
Right-click in Tree View

### DISCUSSION THREAD

**Discussion by kap42** [dendronhq/dendron#3250] [RFCs/Ideas]
Right-clicking on an entry in the Tree View ought to bring up a menu containing the commands that can be done on a note.

**Comment by hikchoi** [comment]
We are planning to enhance the user experience of the tree view in the very near future and this is definitely on the list!

Here is an existing request for this: https://github.com/dendronhq/dendron/issues/2825

If you have any specific ideas related to this, feel free to add a comment to this issue.
